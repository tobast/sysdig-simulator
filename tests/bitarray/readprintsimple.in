2
1101
0001
