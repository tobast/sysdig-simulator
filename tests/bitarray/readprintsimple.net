INPUT a
OUTPUT z
VAR
	a:4, z:4
IN
z = a
