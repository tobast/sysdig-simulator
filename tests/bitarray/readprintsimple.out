1101
0001
